### DISCUSSION TITLE
WPF Theme Switch (ToggleButton)

### DISCUSSION THREAD

**Discussion by vickyqu115** [dotnet/wpf#8496] [Show and tell]
Hello, my name is Vicky. I am a Chinese developer and video creator active in South Korea.

I am excited to introduce the ThemeSwitch control, a CustomControl created by inheriting from the WPF ToggleButton. This control can be downloaded via a NuGet Package, and the original source code along with a demo project illustrating its usage is available on the GitHub repository.

To share my technical knowledge with a wider audience, I have made my content available on GitHub, YouTube, BiliBili, and CodeProject. My goal is to release one piece of content every week (although it's quite challenging).

I hope you find it interesting and enjoyable. Thank you.

----
- [CodeProject](https://bit.ly/468SYy3)
- [Youtube Tutorial](https://bit.ly/3uBkFlQ)
- [BiliBili Tutorial](https://www.bilibili.com/video/BV1ez4y1N7B8/?spm_id_from=333.999.0.0)
- [GitHub Repository](https://github.com/vickyqu115/themeswitch)

---

![image](https://github.com/dotnet/wpf/assets/101777355/9f7c738e-e13f-4aa5-bc37-400ac6f7eb82)

![image](https://github.com/dotnet/wpf/assets/101777355/0c312aaa-d811-4fe5-965c-8c8ae9cab5f0)

**Comment by lindexi** [comment]
Look good to me. Thank you for sharing.
